hooks/pre-push

```diff
@@ -4,16 +4,24 @@
 # Copy and paste it into your `projectDirectory/.git/hooks`
 # Make sure the file is executable by running `chmod +x pre-push
 
-CMD=`./gradlew lintDebug`
 # Check if we actually have commits to push
 commits=`git log @{u}..`
+
 if [ -z “$commits” ]; then
  exit 0
 fi
-$CMD
-RESULT=$?
-if [ $RESULT -ne 0 ]; then
- echo “failed $CMD”
+
+SIZE=1
+while read -r line
+do
+    echo "$line"
+    size=$(($SIZE + ${#line}))
+done < <(./gradlew lintDebug)
```

Comment:
Should this also run unit tests or are we trusting the pusher? 

Could be `./gradlew lintDebug testDebug`